hummingbot/connector/exchange/kraken/kraken_exchange.pyx

```diff
@@ -192,10 +179,12 @@ cdef class KrakenExchange(ExchangeBase):
             self._last_userref = max(int(value["userref"]), self._last_userref)
         self._in_flight_orders.update(in_flight_orders)
 
-    async def asset_pairs(self) -> Dict[str, Any]:
+    async def get_asset_pairs(self) -> Dict[str, Any]:
         if not self._asset_pairs:
             client = await self._http_client()
-            asset_pairs_response = await client.get(ASSET_PAIRS_URI)
+            url = f"{CONSTANTS.BASE_URL}{CONSTANTS.ASSET_PAIRS_PATH_URL}"
+            async with self._throttler.execute_task(CONSTANTS.ASSET_PAIRS_PATH_URL):
+                asset_pairs_response = await client.get(url)
```

Comment:
Is it ok to not reuse `_api_request` to make this http request?

Reply:
Added the use of `_api_request` to standardize the call